### PR TITLE
Issue #31 Use postgresql instead of h2

### LINKED ISSUE #31: Use postgresql instead of h2
We should not be using h2 for the docker compose stack. Switch to Postgres.

@Christian-TD fyi

### PR DESCRIPTION
**Pull request by Christian-TD**
- Removed jbpm
- Removed gogs
- Added postgresql support

closes #31

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
+    restart: always
+    environment:
+      POSTGRES_DB: "aoa"
+      POSTGRES_USER: "aoa-user"
+      POSTGRES_PASSWORD: "passw0rd"
     ports:
-      - "3000:3000"
+      - '5432:5432'
+
+#  gogs:
```
Comment (willfleury) [inline docker-compose.yaml:16]: I'd delete it completely. we can look at git history if we want it in future

File: docker-compose.yaml [hunk 2]
```diff
@@ -49,8 +65,9 @@ services:
       - /tmp/scheduler/dags/:/app/aoa/dags
     privileged: true
     depends_on:
-      - gogs
+#      - gogs
```
Comment (willfleury) [inline docker-compose.yaml:68]: remove

**Review by willfleury — APPROVED** [review]

File: docker-compose.yaml (hunk 2 above)
Comment (Christian-TD) [reply docker-compose.yaml]: done

File: docker-compose.yaml (hunk 1 above)
Comment (Christian-TD) [reply docker-compose.yaml]: done

**Pull request merged**